source/features/repo-wide-file-finder.tsx

```diff
@@ -5,14 +5,23 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import {buildRepoURL} from '../github-helpers';
 import getDefaultBranch from '../github-helpers/get-default-branch';
+import GitHubURL from '../github-helpers/github-url';
+
+async function getBranchForFind(): Promise<string> {
+	try {
+		return (new GitHubURL(location.href)).branch;
+	} catch {
+		return getDefaultBranch();
+	}
+}
```

Comment:
I don’t think we need this, there’s already a “get current branch” function somewhere

Reply:
getCurrentCommitsh

Reply:
Thanks! Updated to use `getCurrentCommitish`